### PR TITLE
Adding VSCode and IntelliJ local directories to gitignore

### PR DESCRIPTION
**Pull request by autodidaddict**
Per a suggestion from another pull request, I'm suggesting these changes as a separate PR. Rumor has it there are people who actually use something other than `vim` to edit code.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,6 @@ npm-debug.log
 yarn-error.log
 testem.log
 .env
+.idea/
+.vscode/
+
```
Comment (vignesh-sankaran) [inline .gitignore]: Could you remove the blank line at the bottom of this file?

**Comment by vignesh-sankaran** [comment]
Think we're good to merge this @carols10cents, thanks @autodidaddict :)

**Comment by autodidaddict** [comment]
I'm going to blame that trailing blank line on the elbows of the person who was next to me on the plane :)

**Comment by carols10cents** [comment]
I think this is more appropriate for each developer to set in their global gitignore file for the editors they use: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

**Pull request closed without merging**